build/commands/lib/config.js

```diff
@@ -402,43 +390,48 @@ Config.prototype.buildArgs = function () {
   let versionParts = version.split('+')[0]
   versionParts = versionParts.split('.')
 
+  const is_component_build = this.isComponentBuild() ? true : undefined
+  const is_official_build = this.isOfficialBuild() ? true : undefined
+
   let args = {
     'import("//brave/build/args/brave_defaults.gni")': null,
-    is_asan: this.isAsan(),
-    enable_full_stack_frames_for_profiling: this.isAsan(),
-    v8_enable_verify_heap: this.isAsan(),
-    is_ubsan: this.is_ubsan,
-    is_ubsan_vptr: this.is_ubsan,
-    is_ubsan_no_recover: this.is_ubsan,
-    is_msan: this.is_msan,
-    safe_browsing_mode: 1,
-    // TODO: Re-enable when chromium_src overrides work for files in relative
-    // paths like widevine_cmdm_compoennt_installer.cc
-    // use_jumbo_build: !this.officialBuild,
-    is_component_build: this.isComponentBuild(),
+    is_msan: this.is_msan != null ? this.is_msan : undefined,
```

Comment:
I'm not sure if I understand why this is needed. `is_msan` was already not being passed on to `args_generated.gni` as it was because undefined/null values get excluded from being emitted. Why is this dance necessary? 

I noticed this is also done down below with:

```
  if (this.is_ubsan != null) {
    args.is_ubsan = this.is_ubsan
    args.is_ubsan_vptr = this.is_ubsan
    args.is_ubsan_no_recover = this.is_ubsan
  }
```

Reply:
oic what you're saying here, after removing `this.isAsan()` it wasn't necessary there either, but I still wanted to group them together logically. I'm fine switching it back though

Reply:
Grouping them together is fine, but I don't think we need to guard them behind an if for `!= null`.